### DISCUSSION TITLE
Regarding the Selection of right Training data from AIMD results.

### DISCUSSION THREAD

**Discussion by jaejae9804** [deepmodeling/deepmd-kit#3481] [General]
Hey everyone,
I want to give my thanks to those who read or reply on my discussion in advance.

I've been working on the stress-strain behavior of Ti2N monolayer(MXene) by using 2nn-meam potential on LAMMPS and I wanted to try using deepmd-kit to get better results. The problem is getting the right training data for my system.
The NNP Im trying to get should be able to predict the stress-strain behavior while an uniaxial tensile stress is applied at 300K. For that, firstly Ive tried using the AIMD results (OUTCAR) at 300K & 400K(2000step, 1fs for each) with the usage of nvt ensemble.  By using these data for training, of course the md results that were applied with this potential was wery inaccurate, since my training data did not include any frames with high external tensile pressure. 
For that, Ive tried adding AIMD results with various external pressure applied along x-axis at 300K(for bulk and 2d system both), and indeed I was able to get a better model that can predict the stress-strain behavior of  my system.  However, still the results such as elastic constants and Young's modulus are not what I desire(far lower than expected).  

My question is, what does the right training data mean exactly?
For example, if I were to use NNP on MD simulation with a system of hydrostatic pressure being applied at 300K, should the training data include AIMD datas  of 300K~400K and hydrostatic pressure being applied?
Should the traing data be closely related to what Im intending to do on MD simulation?

Any help will be greatly appreciated.

**Comment by njzjz** [comment]
I would suggest first checking to see if the accuracy of the AIMD is high enough. See #2704
